plugin-gestures/src/test/java/com/mapbox/maps/plugin/gestures/GesturesPluginTest.kt

```diff
@@ -640,6 +675,30 @@ class GesturesPluginTest {
     verify { cameraAnimationsPlugin.easeTo(any(), any()) }
   }
 
+  @Test
+  fun verifyRotateWithSimultaneousRotateAndPinchToZoomEnabled() {
+    val bearingAnimator = mockk<ValueAnimator>(relaxUnitFun = true)
```

Comment:
we need to mock anchor animator now here as well I guess

Reply:
The reason we mock bearing animator is to capture the `endListenerSlot`, so no need to mock anchor animator